src/views/CreateShareStructure.vue

```diff
@@ -26,7 +26,7 @@
         of shares, each class must be assigned an identifying name such as Class A, Class B, etc.
       </p>
       <div id="share-structure-sample">
-        <ListShareClass :shareClasses="sharesHelpSample"/>
+        <ListShareClass :shareClasses="sharesHelpSample" :isSummary="true"/>
```

Comment:
Just curious, if you just passed `isSummary`, would it work the same? I know this is a Prop that we created, so i'm very curious if work the same as some of the attributes built into Vuetify, that are booleans.  No real action required here :)

Reply:
I tried that in Summary. It dint work like the built in attribute and the component was displayed in the edit mode

Reply:
Yeah, Vuetify is probably checking something on its props that we're not, which is why it works for their components but not ours.
This can be someone's pet project to figure out (hint-hint Cameron).

Reply:
Very cool, thanks for exploring that at least :) 

I'd love too, just need more timmmmeee 😩